tempesta_fw/sock.c

```diff
@@ -440,6 +440,13 @@ ss_do_close(struct sock *sk)
 	}
 }
 
+static void
+ss_do_close(struct sock *sk)
+{
+	__ss_do_close(sk);
+	sock_put(sk);
+}
+
```

Comment:
Why can't we just call ss_close() which does sock_put()? Why do we need 3 functions for socket closing?